### ISSUE TITLE
Stuck on Mac

### ISSUE THREAD

**Issue by HagaiAudax** [ChurchApps/FreeShow#450]
I tried to email the email found on the website but it seems to not be working 

Thank you for freeshow it has been amazing for the most part. 

I have been using it for 4 weeks until last Sunday it broke down, badly in the middle of the Service.

I was using freeshow this Sunday when suddenly I wasn't able to move around my Mac when using freeshow. I couldn't click anything outside of freeshow. 

I tried to shutdown my mac it took some time and when eventually off and restarted, I opened freeshow again. It opened quickly took me back to the second screen (the black screen). And I wasn't able to get out of that. I can't click on anything else. 

I tried uninstalling it then installing again but still nothing happened. Each time I opened freeshow show it goes to another screen and I can move or do anything.

**Comment by HagaiAudax** [comment]
Am using the updated version. 1.1.4

Today I have tried it again same issue. But after showing the blank screen again. I tried clicking it clicked and showed close. Every where on that black screen if it change to a hand cursor I can click (though I can't see anything) and it should close. 

Eventually I tried CMD + Q it has worked today. I have launched freeshow again and now it opened without the going to the second screen. Which is a good step. 


But I can't move around to other apps while using it. I can move around free shows but can't access anything else on my mac. 

![20240323_191544](https://github.com/ChurchApps/FreeShow/assets/164705387/6451e3c9-4d58-459f-ba6e-b7db0fe5285a)

**Comment by vassbo** [comment]
Weird, I will investigate.

**Comment by vassbo** [comment]
What mail address did you send to?

**Comment by vassbo** [comment]
Does this still happen in the latest update?

**Comment by HagaiAudax** [comment]
I haven't updated it yet but i will soon. 

Not sure this is the perfect conclusion but this is what i believe brought the problem - AUTO SAVE. I turned autosave ON.

But this is what i did. I reverted to 1.1.2 version but there was still the same problem though at this time it was a bit slow between the transition. I could see it moving to the display screen. I didn't accept the notifications so i could time then prompt to accept notifications. This time when i started it again, i clicked on the notifications and there it seemed a bit stable (by this time it was automatically updated to 1.14), then i turned OFF auto save, and was able to turn off the output screen. I saved it and closed. Then restarted and everything seemed to be fine.

Even now i working fine after turning off auto save. Not sure this was the exact reason for the problem but i tend to believe so.
